packages/rest/src/validation/request-body.validator.ts

```diff
@@ -137,7 +137,11 @@ function validateValueAgainstSchema(
     return;
   }
 
-  const validationErrors = validate.errors;
+  let validationErrors = validate.errors as AJV.ErrorObject[];
+
+  if (options.ajvTranslate) {
+    validationErrors = options.ajvTranslate(validationErrors);
+  }
 
   /* istanbul ignore if */
   if (debug.enabled) {
```

Comment:
The next line prints a debug log with `util.inspect(validationErrors)`.

In your current proposal, these debug logs will contain translated error message. This is nice when there is a single language involved only and the developer of the application understand the language used for messages returned to clients.

It does not work well for applications supporting multiple languages - for example, if I was developing an app available in English, Czech and Chinese, then I won't be able to understand debug logs translated to Chinese. Personally, I'd prefer to see all debug logs in English using the original message reported by AJV, before any translation or other transformations is applied.

Please run the transformation after the debug log is printed.

If you feel it's useful to see translated messages in the debug log too, then add a new debug log that's printed after the AJV errors were modified.

```ts
if (options.ajvTranslate) {
  validationErrors = options.ajvTranslate(validationErrors);
  /* istanbul ignore if */
  if (debug.enabled) {
    debug(...);
  }
}
```